src/sbt-test/sbt-reactive-app/lagom-endpoints/hello-impl/src/main/scala/HelloServiceImpl.scala

```diff
@@ -0,0 +1,12 @@
+package lagomendpoints.impl
+
+import lagomendpoints.api.HelloService
+import com.lightbend.lagom.scaladsl.api.ServiceCall
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+class HelloServiceImpl(implicit ec: ExecutionContext) extends HelloService {
+  override def hello(id: String) = ServiceCall { _ =>
+    Future { "Hello, " + id }
```

Comment:
This spawns a `Future` on a different thread which is typically not what you want for these scenarios. If you already have a value and you wish to wrap it in a `Future`, use `Future.successful`, i.e. `Future.successful("Hello, " + id)`.

You can also use [string interpolation](https://docs.scala-lang.org/overviews/core/string-interpolation.html) for the string, i.e. `Future.successful(s"Hello $id")`